frontend/sass/styles.scss

```diff
@@ -0,0 +1,9 @@
+$main-body-background-clr: #fcfcfc;
+
+
+@import "normalize";
+@import "partials/base";
```

Comment:
import normalise imports all the normalize, it knows where it lives because i set a sass path in Gruntfile.js, the second one is just a partial import of a file that lives in partials folder and is called base. So SASS let's you break down your sass into smaller files and import them in